doc_k8s/windows/docker/stark-start/Dockerfile-start

```diff
@@ -0,0 +1,27 @@
+FROM node:latest AS build-stage
+
+MAINTAINER NationalBankBelgium - BOURRIER Francois
+
+WORKDIR /usr/projet/
+
+RUN apt-get update -y -q && apt-get install git-sh rsync -y && apt-get upgrade -y -q
```

Comment:
I think it makes more sense to run the upgrade before the install of git-sh and rsync.